lib/interpreters/split_clustering.rb

```diff
@@ -257,14 +260,23 @@ def self.split_solve_core(service_vrp, job = nil, &block)
         # RECURSIVELY CALL split_solve_core AND MERGE SOLUTIONS
         solutions = sides.collect.with_index{ |side, index|
           service_vrp[:split_level] = split_level + 1
+          service_vrp[:split_denominators] << 2**service_vrp[:split_level]
```

Comment:
I am not sure if this would work. `split_solve` is a recursive function and `split_level` increases, decreases, re-increases many times during the optimization process (if my memory servers me well) so this would make the bar  jump back and fort in a big optimization with multiple levels. Plus some branches dive "deeper" (more levels) than some other branches so the denominator calculation is not correct :( (The problem is even worse for dicho)

Lets base the calculation on services_seen(planned+unplanned)/total_services, this has multiple advantages:
- it is correct from the time point of view
- it never jumps back and fort
- the same logic works for dicho, split, repeats, several_solutions, etc.

And instead of saying `Handling sub-process X of Y`, we can say `X services out of Y are handled`. 

Note that the first one gives no information what so ever about the "completion" level -- it might say 1 out of 10 subproblems solved -- i.e., 10% but we might have 100X more time to go or may be we have completed half of the optimization by solving this first sub-problem.

Reply:
@braktar do we need to discuss this subject?

Reply:
The way I implement the avancement using the denominators and the sides didn't not make the bar jump back and forth.

I would like to keep the current logic based on the number of sub_problems processed and keep the progression by planned for the next rework.

Still, there is some issues with the current implementation I need to fix these first before further discussions and review :slightly_smiling_face:

Reply:
Deal :handshake:  ping me when you want me to review

Reply:
Also for the bar to jump back and fort you need a big optimisation so you might want to try with (i) either a big file (ii) or a reasonably sized file with a very small max_split_size..